webpack/scenes/ContentViews/Details/Filters/CVContainerImageFilterContent.js

```diff
@@ -140,16 +155,6 @@ const CVContainerImageFilterContent = ({
             actionButtons={hasPermission(permissions, 'edit_content_views') &&
               <>
                 <Split hasGutter>
-                  <SplitItem>
-                    <Button
-                      ouiaId="add-content-view-container-image-filter-button"
-                      onClick={() => setModalOpen(true)}
-                      variant="primary"
-                      aria-label="add_filter_rule"
-                    >
-                      {__('Add filter rule')}
-                    </Button>
-                  </SplitItem>
                   <SplitItem>
```

Comment:
This shouldn't be entirely remove. It should render conditionally if it's not an empty state.

Reply:
Oh, I thought we want that button to be rendered  only if we have emptyState. Therefore, I rendered the button inside of emptyState